From 861c6cea9ff4e58386a5aae12ba5068376c4c740 Mon Sep 17 00:00:00 2001
From: Nelson A. de Oliveira <naoliv@debian.org>
Date: Fri, 24 Sep 2010 22:12:12 -0300
Subject: [PATCH] Make dget understand -k option (#597988)

Make dget also understand -k option, as a short alternative to --insecure
---
 scripts/dget.pl |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/dget.pl b/scripts/dget.pl
index 57bc974..c6be51d 100755
--- a/scripts/dget.pl
+++ b/scripts/dget.pl
@@ -74,7 +74,7 @@ It is capable of downloading several packages at once.
    --path DIR      Check these directories in addition to the apt archive;
                    if DIR='' then clear current list (may be used multiple
                    times)
-   --insecure      Do not check SSL certificates when downloading
+   -k, --insecure  Do not check SSL certificates when downloading
    --no-cache      Disable server-side HTTP cache
    --no-conf       Don\'t read devscripts config files;
                    must be the first option given
@@ -404,7 +404,7 @@ GetOptions(
     "x|extract"  =>  sub { $dget_unpack = 1 },
     "u|allow-unauthenticated"
                  =>  sub { $dget_verify = 0 },
-    "insecure"   =>  \$opt->{'insecure'},
+    "k|insecure" =>  \$opt->{'insecure'},
     "no-cache"   =>  \$opt->{'no-cache'},
     "noconf|no-conf"   =>  \$opt->{'no-conf'},
     "path=s"     =>  sub {
@@ -574,7 +574,7 @@ directories listed will be searched; hence, the above example could
 have been written as: "--path /srv/pbuilder/result --path
 /home/cb/UploadQueue".
 
-=item B<--insecure>
+=item B<-k>, B<--insecure>
 
 Allow SSL connections to untrusted hosts.
 
-- 
1.7.2.3

